tasks/agent.py

```diff
@@ -934,12 +936,13 @@ def version(
 
     version = get_version(
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Consistent ordering between `version()` and `get_version()` would be nice.

Reply:
Indeed, I was worried that it would require changing all the callsites, but I forgot about named parameters :facepalm: